external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -81,7 +79,6 @@ class DenseHyperparameters(Hyperparameters):
         default_factory=TrainingLoopConfig
     )
     loss: LossConfig = LossConfig(scaling="standard", loss_type="mse")
-    save_model_checkpoints: bool = False
```

Comment:
As long as you're sure this breaking API change is not important.

Is it easy enough to figure out how to set these callbacks for use in config? Wherever we document that argument, the documentation should probably be updated to say it can include any option registered using your new registration function.